### DISCUSSION TITLE
Keep specific view mounted when off screen (DnD)

### DISCUSSION THREAD

**Discussion by gregbzh** [inokawa/virtua#414] [Q&A] [answered]
Hello, is there any option to keep a view mounted (and visible) even if it goes off screen? I am trying to keep a dragged view always visible. Im not sure about the sticky option cause I need the dragged view to follow the dragging positon.

Same issue is described here:

https://github.com/clauderic/dnd-kit/issues/526

Best.

**Comment by inokawa** [comment, marked as answer]
We are planning to add a way to keep items offscreen. It will be tracked in #181 and #370 .

**Reply by mariia-mitiureva** [reply]
@inokawa I need to keep offscreen items mounted for a different reason. Some items can contain video or audio, and if it's playing, it should keep playing when the item moves off the screen.
Will it be resolved in the mentioned tickets as well?

**Reply by inokawa** [reply]
Yes, keeping native behavior at any index(ex. css sticky, selection, video, etc.) is in the scope.

**Reply by mariia-mitiureva** [reply]
I also noticed that in the current version off-screen items within overscaned range are mounted multiple times.

Here is a recording: https://github.com/inokawa/virtua/assets/37265338/e659c7a8-205a-4a87-b22f-5eb96a164ef9
and codesandbox to reproduce: https://codesandbox.io/p/sandbox/virtua-reverse-resize-hn39kp?file=%2Fsrc%2FApp.js%3A20%2C28-20%2C33

In my project an item can contain an input, which is autofocused on mount, and it's automatically scrolled into view when focused. It causes issues like on the recording - textarea is automatically scrolled into view once I stop scrolling:
https://github.com/inokawa/virtua/assets/37265338/3b9164f1-b674-46c9-95f6-6667d618b1eb
